### ISSUE TITLE
no_link_to_reused_ref results in some HTML tabs in JS template not working

### ISSUE THREAD

**Issue by stevespringett** [coveooss/json-schema-for-humans#123]
Executing with the `no_link_to_reused_ref` config option, some UI elements do not work as expected.

```
generate-schema-doc --config no_link_to_reused_ref test.schema.json index.html
```

I'm using the following test schema:
```json
{
  "$schema": "http://json-schema.org/draft-07/schema#",
  "$id": "test.schema.json",
  "type": "object",
  "properties": {
    "objectA": {
      "type": "object",
      "properties": {
        "signature": {
          "$ref": "#/definitions/signature",
          "title": "Signature"
        }
      }
    },
    "objectB": {
      "type": "object",
      "properties": {
        "signature": {
          "$ref": "#/definitions/signature",
          "title": "Signature"
        }
      }
    }
  },
  "definitions": {
    "signature": {
      "$ref": "jsf-0.82.schema.json#/definitions/signature",
      "title": "Signature"
    }
  }
}
```
Note: `jsf-0.82.schema.json` can be found here: 
https://github.com/CycloneDX/specification/blob/v1.4-dev/schema/jsf-0.82.schema.json - Copy it locally to test.

In the following example, objectA works as expected, objectB does not.

![Kapture 2021-12-29 at 18 45 10](https://user-images.githubusercontent.com/3878933/147713184-60e45a19-3841-4d68-9cc1-2c525a630af2.gif)

Expected behavior would be that optionB would behave identical to optionA.

**Comment by dblanchette** [comment]
Hi, I am unable to reproduce the issue and have a fix. Unfortunately, the fix breaks generation when there are recursive references. I should be able to have a complete fix later this week.

**Comment by stevespringett** [comment]
Nice - thanks for the response and all the hard work on this project.

**Comment by dblanchette** [comment]
Should be fixed in 0.39.5.

I hope this is the last time there is a recursivity problem (famous last words)

**Comment by stevespringett** [comment]
Confirmed it's working. Thanks so much for the fix.
